packages/eslint-config-terra/eslint.config.js

```diff
@@ -0,0 +1,83 @@
+module.exports = {
+  extends: 'airbnb',
+  plugins: [
+    'react',
+    'react-hooks',
+    'jsx-a11y',
+    'import',
+    'compat',
+  ],
+  env: {
+    browser: true,
+    jest: true,
+  },
+  rules: {
```

Comment:
with this being a MVB are there any rules we need to evaluate? `react-hooks/exhaustive-deps` from warn to error? @StephenEsser  did you want to fail on `no-multiple-empty-lines` now?

Reply:
My gut says to keep this as a warning because it's a more harmless thing, but I also think it's been a warning long enough we could fail on this MVB. If we want this to be a lint rule and we want to enforce it, it should fail.

+1 to failing (I think this rule can be auto-fixed 99% of the time).

There are two more configs that can be set with this rule:

- "max" (default: 2) enforces a maximum number of consecutive empty lines.
- "maxEOF" enforces a maximum number of consecutive empty lines at the end of files.
- "maxBOF" enforces a maximum number of consecutive empty lines at the beginning of files.

https://eslint.org/docs/rules/no-multiple-empty-lines

The one I'm interested in is potentially setting the max new lines at the start of the file to 0. Unless anyone knows of a conventional reason to start files with new lines.

Reply:
+1 to enforcing

Reply:
I'm going to enforce both multiple empty lines, 1 max, 1 maxEOF, and 0 maxBOF.  Also I'm erroring on exhaustive-deps.